aspects/rules/scala/scala_info.bzl

```diff
@@ -12,38 +12,24 @@ def find_scalac_classpath(runfiles):
             result.append(file)
     return result if found_scala_compiler_jar and len(result) >= 2 else []
 
-def extract_scala_toolchain_info(target, ctx, output_groups, **kwargs):
-    runfiles = target.default_runfiles.files.to_list()
-
-    classpath = find_scalac_classpath(runfiles)
-
-    if not classpath:
-        return None, None
-
-    resolve_files = classpath
-    compiler_classpath = map(file_location, classpath)
-
-    if (is_external(target)):
```

Comment:
Indeed, my version broke the no-build-sync experience here.

From my observation, however, it was sufficient to add `_scalac` back into `PRIVATE_COMPILE_DEPS`. Perhaps `extract_jvm_info` collects the relevant data already?
Anyway, just to be safe, I added this snippet to the `extract_scala_info` as well. Please verify if it's right.

Reply:
hmm im not sure about the `_scalac` but for me it can stay, indeed `extract_jvm_info` should collect relevant data but there might be cases when it wont we triggered

Reply:
Welll, it must've been my mistake during testing, as "now" it seems it indeed works without `_scalc` on the list. In such case I opt for removing it to avoid any superfluous actions.